UnityProject/Assets/Scripts/Sprite Handler/AnimationInfo.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AnimationInfo : MonoBehaviour
```

Comment:
This class would be better off as a ScriptableObject. Have a look at the guidance in #2311 and try to follow it's advice.